src/client/Client.cc

```diff
@@ -9283,8 +9283,8 @@ int Client::_read_sync(Fh *f, uint64_t off, uint64_t len, bufferlist *bl,
 
   ldout(cct, 10) << __func__ << " " << *in << " " << off << "~" << len << dendl;
 
-  ceph::mutex flock = ceph::make_mutex("Client::_read_sync flock");
-  ceph::condition_variable cond;
+  //ceph::mutex flock = ceph::make_mutex("Client::_read_sync flock");
+  //ceph::condition_variable cond;
```

Comment:
These lines should just be deleted.

Reply:
it has been modified.please review, thanks.